Marlin/Regulator.cpp

```diff
@@ -9,9 +9,45 @@
 
 #if ENABLED(E_REGULATOR)
 
-// Set the output pressure (in psi) of the regulator.
-void setOutputPressure(float desired_pressure) {
+//===========================================================================
+//=============================== Definitions ===============================
+//===========================================================================
+
+// Relevant Pins
+// Voxel8 C2
+// #define PNEUMATIC_PUMP_PIN    7   // Pump driver
+// #define PNEUMATIC_PIN         5   // A5 Input analog tank pressure reading
+// #define PNEUMATIC_CART1_PIN   4   // A4 Cartridge 2 analog pressure reading
+
```

Comment:
Good call, I pretty much left it there for myself haha but it's not super portable / relevant anymore.